test/integration/testStreamRows.js

```diff
@@ -462,3 +464,91 @@ describe('Test Stream Rows HighWaterMark', function ()
     });
   });
 });
+
+describe('Test stream row modes', function () {
+  this.timeout(18000);
+  let connection;
+  const sql = 'select * ' +
+    'from (select \'a\' as key, 1 as foo, 3 as name) as table1 ' +
```

Comment:
Let's add a test for more than two instances of the same name.

Reply:
Ok